packages/suite-web/vite.config.ts

```diff
@@ -0,0 +1,160 @@
+import { viteCommonjs } from '@originjs/vite-plugin-commonjs';
+import react from '@vitejs/plugin-react';
+import { execSync } from 'child_process';
+import { readdirSync } from 'fs';
+import { resolve } from 'path';
+import { Plugin, ViteDevServer, defineConfig } from 'vite';
+import { nodePolyfills } from 'vite-plugin-node-polyfills';
+import wasm from 'vite-plugin-wasm';
+
+import { suiteVersion } from '../suite/package.json';
+import { assetPrefix, project } from '../suite-build/utils/env';
+
+// Plugin to serve static files with /static prefix
+const staticAliasPlugin = (): Plugin => ({
+    name: 'static-alias',
+    configureServer(server: ViteDevServer) {
+        // Middleware to handle /static requests
+        server.middlewares.use((req, res, next) => {
+            if (req.url?.startsWith('/static/')) {
+                // Rewrite the URL to access the file from the public directory
+                req.url = req.url.replace('/static/', '/');
+            }
+            next();
+        });
+    },
+});
+
+// Plugin to serve core.js in dev mode
+
+// Plugin to handle workers similar to webpack's worker-loader
+const workerPlugin = (): Plugin => ({
+    name: 'worker-loader',
+    transform(code, id) {
```

Comment:
nit: unused variable
```suggestion
    transform(_code, id) {
```

Reply:
<3 adding